exercises/05.actions/05.solution.history-revalidation/server/app.js

```diff
@@ -35,31 +55,32 @@ app.use((req, res, next) => {
 		next()
 	}
 })
+*/
```

Comment:
@kentcdodds I really don't understand this part.

Reply:
The url can be: `/someId?search=` and I want that to change to `/someId` just to keep it clean.